src/OpenTelemetry.Exporter.Stackdriver/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 * Update OTel SDK version to `1.5.0`.
```

Comment:
You could probably just update this entry as this package hasn't been published with a `1.5.0` dependency.

Reply:
Same for EventCounters.